Gemfile

```diff
@@ -2,12 +2,21 @@ source 'https://rubygems.org'
 
 gemspec
 
+git_source(:github) { |repo_name| "https://github.com/#{repo_name}" }
+
 gem 'rake', '~> 12.0'
 gem 'rspec', '~> 3.0'
 
+gem 'dry-logic', '~> 1.0', github: 'dry-rb/dry-logic', branch: 'master'
+gem 'dry-types', '~> 1.0', github: 'dry-rb/dry-types', branch: 'master'
+gem 'dry-struct', '~> 1.0', github: 'dry-rb/dry-struct', branch: 'master'
```

Comment:
H'm, looks like rom-factory doesn't depend on these three gem. What for it is here?

Reply:
@v-kolesnikov because rom from master needs them